gulp-tasks/utils/constants.js

```diff
@@ -23,4 +23,11 @@ module.exports = {
 
   // This is the environments that we should use for NODE_ENV.
   BUILD_TYPES: ['dev', 'production'],
+
+  // Used to identify whether a given package should be built as a Node or
+  // Browser bundle.
+  PROJECT_TYPES: {
```

Comment:
Can we just add this information to the package.json under "workbox". Reading through the above code, this has added a lot of boilerplate and I'm still not sure how packages are determined as node or browser packages.

Reply:
Sure, that sounds like a good approach. I'll update the existing `package.json` files to reflect that they're targeted to the browser.